chart/tests/test_airflow_common.py

```diff
@@ -248,3 +248,23 @@ def test_have_all_variables(self):
             assert variables == jmespath.search(
                 "spec.template.spec.containers[0].env[*].name", doc
             ), f"Wrong vars in {component}"
+
+    def test_have_all_config_mounts_on_init_containers(self):
+        docs = render_chart(
+            values={},
+            show_only=[
+                "templates/scheduler/scheduler-deployment.yaml",
+                "templates/workers/worker-deployment.yaml",
+                "templates/webserver/webserver-deployment.yaml",
+                "templates/triggerer/triggerer-deployment.yaml",
+            ],
+        )
+        assert 4 == len(docs)
+        for doc in docs:
+            expected_mount = {
+                "subPath": "airflow.cfg",
+                "name": "config",
+                "readOnly": True,
+                "mountPath": "/opt/airflow"
+            }
```

Comment:
```suggestion
        expected_mount = {
            "subPath": "airflow.cfg",
            "name": "config",
            "readOnly": True,
            "mountPath": "/opt/airflow"
        }
        for doc in docs:
```
nit: don't need to set `expected_mount` each loop.